### PR TITLE
setup project structure for frontend and backend

### PR DESCRIPTION
**Pull request by Orest-Haponov**

### CODE REVIEW

**Comment by Orest-Haponov** [comment]
@nikita-shmatko @omnomnomka 
- Changed backend build from express to nest.js;
- Stabilized dependencies;
- Moved all @types to devDependencies;
- Added scss;
- Updated readme and gitignor

File: frontend/.eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
..."results":"11","hashOfConfig":"7"},{"filePath":"12","messages":"13","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},"101bmcb",{"filePath":"15","messages":"16","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},{"filePath":"17","messages":"18","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},{"filePath":"19","messages":"20","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},{"filePath":"21","messages":"22","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"C:\\sigma-js-internship\\frontend\\src\\index.tsx",[],["23","24"],"C:\\sigma-js-internship\\frontend\\src\\serviceWorker.ts",[],"C:\\sigma-js-internship\\frontend\\src\\app\\App.tsx",[],"C:\\sigma-js-internship\\frontend\\src\\store\\store.ts",[],"C:\\sigma-js-internship\\frontend\\src\\store\\reducer\\rootReducer.ts",[],{"ruleId":"25","replacedBy":"26"},{"ruleId":"27","replacedBy":"28"},"no-native-reassign",["29"],"no-negated-in-lhs",["30"],"no-global-assign","no-unsafe-negation"]
```
Comment (omnomnomka) [inline frontend/.eslintcache]: Please, add this file to gitignore

File: frontend/public/robots.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```
Comment (omnomnomka) [inline frontend/public/robots.txt]: I don't think this file is necessary for our project

**Review by omnomnomka — CHANGES_REQUESTED** [review]

**Comment by Orest-Haponov** [comment]
@omnomnomka @nikita-shmatko 
- removed .eslintcache and robots.txt from project
- updated .gitignore (.eslintcache was already in it, but in some reason didn't work)

**Review by omnomnomka — APPROVED** [review]

**Review by cx-nikita-shmatko — APPROVED** [review]

**Pull request merged**